### PR TITLE
Bugs Fix

### LINKED ISSUE #707: "Ready to import" text should say "Importing cluster" when cluster is being imported, and Import button is disabled
"Ready to import" text should say "Importing cluster" when cluster is being imported, and Import button is disabled -- see screenshot below (note 2nd row where "Import" button is disabled):

screen shot 2017-10-24 at 2 06 11 pm

@gnehapk @a2batic @mcarrano @Tendrl/tendrl-qe

### PR DESCRIPTION
**Pull request by a2batic**
Tendrl-bug-id: 
#671 : Tooltip appears when the button text truncates 
#706 : Password show/hide icon fixed
#707 : Message changed to 'Importing cluster' if the cluster is importing

### CODE REVIEW

File: src/modules/users/add-user/add-user.html [hunk 1]
```diff
@@ -59,10 +59,10 @@
                 <div class="col-sm-8 col-md-6 inner-addon">
                     <input type="{{ addUserCntrl.confirmPassword ? 'text' : 'password' }}" data-id="password" aria-describedby="passwordStatus" name="confirmPassword" ng-model="addUserCntrl.user.confirmPassword" minlength="8" required/>
                     <span id="passwordStatus" class="showPassword" ng-click="addUserCntrl.toggleConfirmPassword()">
-                        <span ng-if="addUserCntrl.confirmPassword === 1">
+                        <span ng-if="addUserCntrl.confirmPassword == 1">
```
Comment (gnehapk) [inline src/modules/users/add-user/add-user.html]: @a2batic Replace "==" with "===".

File: src/modules/users/add-user/add-user.html [hunk 2]
```diff
@@ -59,10 +59,10 @@
...
                             <i class="glyphicon glyphicon-eye-open" aria-hidden="true"></i>
                         </span>
-                    <span ng-if="addUserCntrl.confirmPassword === 0">
+                    <span ng-if="addUserCntrl.confirmPassword == 0">
```
Comment (gnehapk) [inline src/modules/users/add-user/add-user.html]: @a2batic same as above comment.

File: src/modules/users/add-user/add-user.html (hunk 1 above)
Comment (a2batic) [reply src/modules/users/add-user/add-user.html]: Acknowledge

File: src/modules/users/add-user/add-user.html (hunk 2 above)
Comment (a2batic) [reply src/modules/users/add-user/add-user.html]: Acknowledge

**Review by gnehapk — APPROVED** [review]

**Pull request merged**